develpoedCalculator.js

```diff
@@ -1 +1,3 @@
-class developedCalculator {};
\ No newline at end of file
+class DevelopedCalculator {
+    developedCalculator = new Calculator();
```

Comment:
DevelopedCalculator needs to extend Calculator class not create an instance of Calculator class 

```js
class DevelopedCalculator extends Calculator {
}

```
Please read more about extending here
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/extends

Reply:
When `DevelopedCalculator` extends `Calculator`, it inherits all the public methods and properties of Calculator class. this mean the instances of `DevelopedCalculator` have render method (they inherits the render method from the `base` class )